manifests/base/templates/enable-mesh-mtls.yaml

```diff
@@ -4,7 +4,6 @@ TODO(https://github.com/istio/istio/issues/18199) remove this configuration from
 
 */ -}}
 
-{{ if .Values.clusterResources }}
 {{- if .Values.global.mtls.enabled }}
```

Comment:
I think in 1.6 we should get rid of those globals - defaults in mesh config or explicit (not installer) user configs.

Reply:
Agree.. @diemtvu @incfly any concerns? I don't think we ever talked about the DestinationRules we are setting by default, are we ok with removing those as well?

Reply:
Yeah. I'm ok with that. We already try to prep this clean up by reducing the use of `global.mtls.enabled` in most of 1.5 docs.

Reply:
destination rule as well?

Reply:
You can remove the whole file. Even better, the flag it self.